haiku-apps/genesis_commander/genesis_commander-0.42.recipe

```diff
@@ -15,6 +18,7 @@ PROVIDES="
 REQUIRES="
 	haiku$secondaryArchSuffix
 	"
+
```

Comment:
omg i do always checking all files multiple times but yeah i can't still see that haha.